### PR TITLE
feature: cpd-782 Cycle Report Updates

### PR DESCRIPTION
**Pull request by itsmostafa**
Update Cycle Report content
change report pdf file names
fix donut graph in cycle reports

## 🧪 Testing ##
tested locally

## ✅ Pre-approval checklist ##

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - *eschew scope creep!*
- [x] *All* future TODOs are captured in issues, which are referenced
      in code comments.
- [x] All relevant type-of-change labels have been added.
- [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [x] All relevant repo and/or project documentation has been updated
      to reflect the changes in this PR.
- [x] Tests have been added and/or modified to cover the changes in this PR.
- [x] All new and existing tests pass.

### CODE REVIEW

**Review by dylanj1752 — APPROVED** [review]
LGTM

File: src/utils/emails.py [hunk 1]
```diff
@@ -111,8 +136,7 @@ def send_mailgun(
             auth=("api", self.sending_profile["mailgun_api_key"]),
             data=data,
             files=[
-                ("attachment", ("report.pdf", open(a, "rb").read()))
-                for a in attachments
+                ("attachment", (filename, open(a, "rb").read())) for a in attachments
```
Comment (xlyk) [inline src/utils/emails.py]: Does this close the file after opening it? Normally I would use a context manager to automatically close files after reading them

**Review by xlyk — APPROVED** [review]

File: src/utils/emails.py (hunk 1 above)
Comment (itsmostafa) [reply src/utils/emails.py]: @xlyk very good point. I'll refactor this portion

**Review by xlyk — APPROVED** [review]
nice work

**Pull request merged**